lib/Doctrine/DBAL/Driver/SQLAnywhere/SQLAnywhereStatement.php

```diff
@@ -73,7 +73,10 @@ class SQLAnywhereStatement implements IteratorAggregate, Statement
     public function __construct($conn, $sql)
     {
         if (! is_resource($conn)) {
-            throw new SQLAnywhereException('Invalid SQL Anywhere connection resource: ' . $conn);
+            throw new SQLAnywhereException(sprintf(
+                'Invalid SQL Anywhere connection resource, "%s" given.',
```

Comment:
Sometimes `GetVariableType` will return a token (class name) and it will be wrapped in double quotes here. Should we move the quoting up in to `GetVariableType` or is that acceptable in this case?

Reply:
We should omit quotes here. The way I see it, we should quote string _values_. Something which would make sense as a value of type string in the code (e.g. 'Configuration parameter "username" is missing' or 'Database "doctrine" does not exist').

In this case, it's not a string `"\Foo\Bar"` or string `"array"`. It's just a word in the sentence.